code/game/world.dm

```diff
@@ -222,8 +222,11 @@ GLOBAL_VAR_INIT(world_topic_spam_protect_time, world.timeofday)
 			if(input["key"] != config.comms_password)
 				return "Bad Key"
 			else
+				var/prtext = input["announce"]
+				if(copytext(prtext, 1, 23) == "Pull+Request+merged+by") // its urlencoded
+					GLOB.pending_server_update = TRUE
```

Comment:
```suggestion
				var/prtext = input["announce"]
				var/pr_substring = copytext(prtext, 1, 23)
				if(pr_substring == "Pull Request merged by")
					GLOB.pending_server_update = TRUE
```
![image](https://user-images.githubusercontent.com/25063394/93378843-e9b33a80-f854-11ea-85ce-2536d62361fa.png)
This configuration works, and I tested it with the script itself